resources/quality/ultimaker_s3/um_s3_cc0.4_um-pla_0.15mm.inst.cfg

```
@@ -0,0 +1,29 @@
+[general]
+definition = ultimaker_s3
+name = Normal - Experimental
+version = 4
+
+[metadata]
+is_experimental = True
+material = ultimaker_pla
+quality_type = fast
+setting_version = 22
+type = quality
+variant = CC 0.4
+weight = -1
+
+[values]
```

Need to enable the acceleration since it is now disabled by default
```suggestion
[values]
_plugin__curaenginegradualflow__0_1_0__gradual_flow_enabled = True
```